### PR TITLE
update travis to xenial

### PR DESCRIPTION
**Pull request by therealssj**
Fixes: #2260 
Update travis dist to `xenial`

Does this change need to mentioned in CHANGELOG.md?
no

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -55,7 +55,9 @@ install:
 
 before_script:
   - if [[ "$TRAVIS_OS_NAME" == "osx" && "$TRAVIS_PULL_REQUEST" == false ]]; then ./ci-scripts/add-key.sh; fi
-  - if [[ "$TRAVIS_OS_NAME" != "osx" ]]; then export DISPLAY=:99.0 && sh -e /etc/init.d/xvfb start; fi
+
+services:
```
Comment (gz-c) [inline .travis.yml]: Why is this changed?

**Comment by gz-c** [comment]
Does not need to be in changelog, its irrelevant to users

File: .travis.yml (hunk 1 above)
Comment (therealssj) [reply .travis.yml]: that's how it works now in xenial. the old `before_script` execution gives error

Comment (therealssj) [reply .travis.yml]: https://changelog.travis-ci.com/ubuntu-xenial-16-04-build-environment-update-86123

Comment (gz-c) [reply .travis.yml]: The old way doesn't run on OS X, the new way does?   You link doesn't mention xvfb
